<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_new_cat</class>
 <widget class="QDialog" name="add_new_cat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add new cat</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="catID_Label">
       <property name="text">
        <string>Cat ID *</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="catID_LineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="impacts_segments_Label">
       <property name="text">
        <string>Impacts segments</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="impacts_segments_LineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="motion_direct_Label">
       <property name="text">
        <string>Direction of movement</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="motion_direct_LineEdit">
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Comment</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QTextEdit" name="comment_textEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox_anc">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox_anc</sender>
   <signal>rejected()</signal>
   <receiver>add_new_cat</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
